Platon/backend/app/search_engine/forms.py

```diff
@@ -0,0 +1,16 @@
+from wtforms import Form, StringField, IntegerField, BooleanField, validators
+from app import api
+from flask_restplus import reqparse
+
+class UserSearchForm(Form):
+    search_query = StringField("search_query",validators=[validators.DataRequired()])
+    job_filter = IntegerField("job_filter")
+    page = IntegerField("page")
+    per_page = IntegerField("per_page")
+
+user_search_parser = reqparse.RequestParser()
+user_search_parser.add_argument('search_query',required=True,type=str,help="Search Query",location='args')
+user_search_parser.add_argument('job_filter',type=int,help="Job Filter(Give as ID)",location='args')
```

Comment:
I'm confused. Android and Frontend cannot give ID of the jobs for filtering. There is no API that returns the ID of a job. How will it be possible?

Reply:
Yes they can. Because actually they have to get all jobs from the endpoint @hilal-demir is implemented. Then they will have ids of jobs. I am not sure about it but if @hilal-demir do not sent such an information, she must change. Because in this situation it is irrelevent to transports jobs as strings. Because a string whose length is bigger than 4 contains more space than an integer.

Reply:
In my Jobs' get endpoint, i am returning the job names, not job ids. I could change it from name to id. Is there anywhere else which frontend will need the list of all jobs as Strings? If so, I think that they can get the ids and get corresponding names from the database.

Reply:
I think it is better to send them both. It can be a list of dictionaries which includes names and ids of all the jobs.

Reply:
I will make the required change in the next commit @hilal-demir.